### ISSUE TITLE
actionMenu items not reading on VO on iOS

### ISSUE THREAD

**Issue by jnurthen** [adobe/spectrum-web-components#759]
### Expected Behaviour
Menu Items should be read out

### Actual Behaviour
only the menu items for Deselect and Active work path are read - others are just read as menuitem

### Reproduce Scenario (including but not limited to)
Run demo page and open the more actions menu

#### Steps to Reproduce
https://opensource.adobe.com/spectrum-web-components/components/action-menu
open more actions menu
swipe right and note that only menuitem is read for menu items 2 to 5

#### Platform and Version
iOS 13
iPadOS 14

**Comment by Westbrook** [comment]
This component extends `Dropdown` under the covers, so I'm going to close this as a duplicate of #764. I just checked this out and am very surprised at this divergence. We'll track it down soon!
